src/groups/mwc/mwcc/mwcc_monitoredqueue.h

```diff
@@ -590,7 +590,7 @@ template <class QUEUE, class QUEUE_TRAITS>
 inline int MonitoredQueue<QUEUE, QUEUE_TRAITS>::pushBack(
     bslmf::MovableRef<ElementType> value)
 {
-    const int currentQueueLen = numElements();
+    const int currentQueueLen = static_cast<int>(numElements());
```

Comment:
Here it looks like we're going from `numElements()` of type `Int64` to `int`, and using it below in a comparison against `capacity()` of type `Int64`.  I think it's best to change the `int` to an `Int64` than lose information between `Int64` and `int`, only to promote the result back to `Int64`.  Like what you did in [mwcc_orderedhashmap.h](https://github.com/bloomberg/blazingmq/pull/245/files#diff-98a2f035d83dbc4efe32843358bbd56f2644179e4afe94ff5304c51e9133e67f).